worlds/stardew_valley/test/TestRules.py

```diff
@@ -307,3 +311,47 @@ def GiveItemAndCheckReachableMine(self, item_name: str, reachable_level: int):
             assert not self.world.logic.can_mine_perfectly_in_the_skull_cavern()(self.multiworld.state)
 
         self.remove(item)
+
+
+class TestRecipeLogic(SVTestBase):
+    options = {
+        options.BuildingProgression.internal_name: options.BuildingProgression.option_progressive,
+        options.SkillProgression.internal_name: options.SkillProgression.option_progressive,
+        options.SeedShuffle.internal_name: options.SeedShuffle.option_shuffled,
+    }
+
+    # I wanted to make a test for different ways to obtain a pizza, but I'm stuck not knowing how to block the immediate purchase from Gus
```

Comment:
Since you have regions setup ... disconnect gus's entrance during the test?

Reply:
Just so the comment doesn't appear to have been ignored, I want to mention that I asked for clarification as to how I can do that, in Discord DMs, because I was not aware this was possible